packages/@spectrum-icons/workflow/scripts/updatedNamesMap.js

```diff
@@ -0,0 +1,6 @@
+
+export const renameMap = {
+  A4uSelectSubtract: 'SelectSubstract',
+  A4uSubtractBackPath: 'SubstractBackPath',
+  A4uSubtractFromSelection: 'SubstractFromSelection'
```

Comment:
So just to be absolutely clear, there is an icon called `SelectSubtract` and an icon called `A4uSelectSubtract`? And thus we need to rename the A4u version to something slightly different when getting rid of the A4u from the icon name?

Just curious why we are calling it `SelectSubstract`, `substract` feels like a typo..

Reply:
substract was a typo, technically there only exists the new `A4uSelectSubtract` now to correct the typo, however, that's breaking, so we have this updatedNamesMap, which will map between the A4u name and a new name I want to give it, in this case, it's the old name that we want to continue to support, and I can just go directly to the name we want, don't need to worry about the prefix.

It looks like I missed those two in the description. They do look correct. Were there others?

Reply:
No those were the only two actually, I was mistaken on a third. Thanks for the typo explaination, lgtm then